### PR TITLE
Index page update

### PR DESCRIPTION
**Pull request by OliaG**

### CODE REVIEW

File: docs/architecture/modernize-desktop/index.md [hunk 1]
```diff
@@ -31,29 +31,65 @@ Mac and macOS are trademarks of Apple Inc.
...
+
+Microsoft's approach to modernizing existing applications is to give you the flexibility to create your own customized path. All the modernization strategies described in this book are mostly independent. You can choose ones that are relevant for your application and skip others that aren't important for you. In other words, you can mix and match the strategies to best address your application needs.
 
 ## Who should use the book
 
-TODO: Add details
+We wrote this book for developers and solution architects who want to modernize existing Windows Forms and WPF desktop applications to leverage the benefits of .NET Core and Windows 10.
+
+You also might find this book useful if you are a technical decision maker, such as an enterprise architect or a development lead/director who just wants an overview of the benefits that you can get by updating existing desktop applications.
```
Comment (gewarren) [inline docs/architecture/modernize-desktop/index.md:74]:
```suggestion
You might also find this book useful if you are a technical decision maker, such as an enterprise architect or a development lead or director who wants an overview of the benefits of updating existing desktop applications.
```

File: docs/architecture/modernize-desktop/index.md [hunk 2]
```diff
@@ -31,29 +31,65 @@ Mac and macOS are trademarks of Apple Inc.
...
+Co-Authors:
 
-TODO: Add authors here.
+Olia Gavrysh, PM, .NET team, Microsoft
+
+Miguel Angel Castejón Dominguez, Innovation Architect, Kabel
+
+Participants and reviewers:
+
+Maira Wenzel, Sr. Content Developer, Microsoft
```
Comment (gewarren) [inline docs/architecture/modernize-desktop/index.md:42]:
```suggestion
Maira Wenzel, Sr. PM, Microsoft
```

**Review by gewarren — APPROVED** [review]

**Review by mairaw — APPROVED** [review]
I've incorporated @gewarren's feedback and expanded the titles like we've been doing in other books such as https://docs.microsoft.com/en-us/dotnet/architecture/cloud-native/

**Comment by mairaw** [comment]
🎆 🎆 🎆

**Pull request merged**